feat: redesign about page

paulatulis opened the pull request:
Lots of commented out stuff to clean up before creating actual PR. Need to sync with Kristen and Chris.

lottamus commented:
Deploy preview for *stoplightio* ready!

Built with commit e28f5610b6eb489ca933fa26af12637b6b30c2b7

https://deploy-preview-1258--stoplightio.netlify.com

ksatx commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Improve /about-us](https://trello.com/c/x2ousw22/252-improve-about-us)

paulatulis commented:
- added mission statement to the top of the 
- added hero photo 
- added Values section - NOTE: we don't have our company values finalized yet, so this is a preview
- added feature strip and section for History, Careers, Social Good
- kept the rest of the page as is (individual team photos, press, businesses, action bar)

![Screenshot 2019-11-19 at 12 38 07 PM](https://user-images.githubusercontent.com/47359669/69175487-7cfed080-0ac9-11ea-88e6-aead88a06b24.png)
![Screenshot 2019-11-19 at 12 38 16 PM](https://user-images.githubusercontent.com/47359669/69175494-7f612a80-0ac9-11ea-9212-dc272f6e1404.png)

ksatx commented on netlify/pages/about.yaml:
```diff
@@ -2,13 +2,76 @@ path: /about
+        nibh volutpat fringilla. Vestibulum risus velit turpis duis.Lorem ipsum dolor sit amet,
+        consectetur adipiscing elit.
+      titleColor: slategrey
+
```
Values flexboxes are displaying too wide across, so the last two are stacking underneath. Can you adjust the width so that they're all in one row on desktop and only stack in a column on mobile?

![values stacked](https://user-images.githubusercontent.com/51092436/69177626-825e1a00-0acd-11ea-97df-f742be6dadab.PNG)

paulatulis replied:
@ksatx i think it depends on how many values we end up with, and how long they are, but the text gets a bit squished when they are all in the same row. Plenty of options if that becomes an issue (I can widen the entire page's margins, or widen the margins of each value card, or shrink the text, etc.) Do you have any thoughts right off the bat?

![Screenshot 2019-11-19 at 3 05 45 PM](https://user-images.githubusercontent.com/47359669/69186399-11732e00-0ade-11ea-9d28-4bc2825829ca.png)

ksatx replied:
First thought off the bat - tell @marbemac  to pick 4 or fewer values, haha. Second thought off that bat, when we get to that point in the road, making some kind of hover toggle. I'll (hopefully) be a little handier with javascript functions by the time we unveil the core values, so I'm willing to volunteer to build it out.
![1](https://user-images.githubusercontent.com/51092436/69189801-a8db7f80-0ae4-11ea-8904-bda53961a782.png)
![2](https://user-images.githubusercontent.com/51092436/69189805-abd67000-0ae4-11ea-998c-086ebaa8c0b5.png)

ksatx replied:
Obviously not in those garish colors, haha, and with font awesome icons, but you get it. @paulatulis

paulatulis replied:
This is cool and something we can play around with while the values are solidified. Sounds good!

ssspear commented on netlify/pages/about.yaml:
```diff
@@ -2,13 +2,76 @@ path: /about
+      image: /images/team/farm.jpg
+      shortName: Social Good
+      title: Social Good
 team:
```
Don't forget to add yourself to the team 🙃

ksatx replied:
Seconded!

ssspear commented on src/components/Value/config.js:
```diff
@@ -0,0 +1,17 @@
+  label: 'Values',
+  name: 'values',
+  widget: 'list',
+  fields: [
```
Are these both going to be required?

paulatulis replied:
Thanks for flagging!

ssspear commented on src/components/FeatureSection/index.tsx:
```diff
@@ -118,7 +118,7 @@ export const FeatureSection: React.FunctionComponent<IFeatureSection> = ({
   return (
     <Section id="product" {...sectionProps} className="pb-32" noPadding>
-      <FeatureStrip features={features} />
+      {/* <FeatureStrip features={features} /> */}
```
Should this be removed, or is it commented out for a reason?

ksatx commented on netlify/pages/about.yaml:
```diff
@@ -2,13 +2,76 @@ path: /about
+    - description: >-
+        At Stoplight, we believe in doing good. That's why we're partnering with [Offset Earth](https://offset.earth/stoplightinc) to offset the carbon footprint of every single 
+        Stoplight employee. This includes our time in and outside of the office. So far, we've 
+        planted 396 trees and removed 59 tons of carbon from the planet.
```
At Stoplight, we believe the positive impact we can make on the world isn't limited to the API space. That’s why we work with organizations like [Offset Earth](https://offset.earth/stoplightinc), [Urban Roots](https://urbanrootsatx.org/), and [OSMI](https://osmihelp.org/). Check out our [Offset Earth Forest](https://offset.earth/stoplightinc) to learn how we offset the carbon footprint of every Stoplight employee and became a climate positive workforce.

ksatx commented on src/components/Value/index.tsx:
```diff
@@ -0,0 +1,42 @@
+    <Section id="values" className="pt-16" noPaddingB>
+      <Container title="Our Values" className="border-b px-0 pb-20 pb-16">
+        <div className="container flex flex-wrap justify-start md:justify-around md:px-0">
+          {values.map((value, key) => (
```
Ditto earlier comment on the "Values" flexbox, but I'm not experienced enough to identify where the problem is.

paulatulis commented:
Commented out values section while we wait to solidify company values. Moved mission statement to below hero image: 

![Screenshot 2019-11-20 at 1 29 22 PM](https://user-images.githubusercontent.com/47359669/69271128-e5b57e00-0b99-11ea-8e0d-f317fabb6877.png)

smf434 commented:
Looking really good!

lottamus commented on netlify/pages/about.yaml:
```diff
@@ -167,7 +232,7 @@ collage:
   ctas:
     - color: accent
-      href: 'https://angel.co/company/stoplight/jobs'
+      href: 'https://stoplight.careerplug.com/jobs'
```
Why are we switching to career plug? IMO angel.co has a much better UI. What kind of customization do we have over that site?

```suggestion
      href: 'https://angel.co/company/stoplight/jobs'
```

paulatulis replied:
Not sure who has final say on this. Doesn't matter to me, but I thought we were moving to Career Plug as main site.

lottamus replied:
Let's keep it as angel.co until we have a better solution. Career plug's UI is not up to our standards lol

lottamus replied:
I've added a ticket to create our own solution: https://github.com/stoplightio/stoplight.io/issues/1271

lottamus commented on src/components/Collage/index.tsx:
```diff
@@ -16,7 +16,7 @@ export const Collage: React.FunctionComponent<ICollage> = ({ images, title, cta,
 
   return (
-    <Section {...sectionProps}>
+    <Section {...sectionProps} noPaddingT>
```
I suggest passing this prop in from wherever you're using it. If you add `noPaddingT` here, it will apply everywhere this component is used

```suggestion
    <Section {...sectionProps}>
```

lottamus commented on netlify/pages/about.yaml:
```diff
@@ -1,14 +1,79 @@
+        Nam ultricies vitae massa a rutrum. Vestibulum iaculis est vel
+        nibh volutpat fringilla. Vestibulum risus velit turpis duis.Lorem ipsum dolor sit amet,
+        consectetur adipiscing elit.
+      titleColor: slategrey
```
Suggest removing this until we have our core values defined

paulatulis replied:
@lottamus is it okay to comment out for now, or would you recommend fully deleting?

lottamus replied:
Delete, once we have content someone can use the Admin portal to enter it back in

lottamus commented on src/components/PressSection/index.tsx:
```diff
@@ -40,7 +40,7 @@ export const PressSection: React.FunctionComponent<IPressSection> = ({ id, title
 
   return (
-    <Section id={id}>
+    <Section id={id} noPaddingB>
```
Same as above, suggest passing this in as a prop and destructuring it like we're doing in the other components

```suggestion
    <Section id={id} {...sectionProps}>
```

lottamus commented on src/components/Value/index.tsx:
```diff
@@ -0,0 +1,42 @@
+export const ValueSection = ({ values }: IValueSection) => {
+  return (
+    <Section id="values" className="pt-16" noPaddingB>
+      <Container title="Our Values" className="border-b px-0 pb-20 pb-16">
```
You have both `pb-20` and `pb-16` which conflict with eachother. Suggest picking one

paulatulis replied:
Yikes! Thanks for flagging.

lottamus commented on src/components/Value/index.tsx:
```diff
@@ -0,0 +1,42 @@
+}
+export const ValueSection = ({ values }: IValueSection) => {
+  return (
+    <Section id="values" className="pt-16" noPaddingB>
```
To be consistent with the other sections

```suggestion
    <Section id="values" {...sectionProps}>
```

lottamus commented on src/components/Value/index.tsx:
```diff
@@ -0,0 +1,42 @@
+  return (
+    <Section id="values" className="pt-16" noPaddingB>
+      <Container title="Our Values" className="border-b px-0 pb-20 pb-16">
+        <div className="container flex flex-no-wrap justify-center md:justify-around md:px-0 sm:flex-wrap">
```
I believe the `<Container>` component adds the `container` classname so you shouldn't need to add it here

```suggestion
        <div className="flex flex-no-wrap justify-center md:justify-around md:px-0 sm:flex-wrap">
```

lottamus commented on src/templates/About/index.tsx:
```diff
@@ -5,10 +5,13 @@ import { withRouteData } from 'react-static';
 import { Layout } from '../../components/Layout';
 import { IPressSection, PressSection } from '../../components/PressSection';
+import { Section } from '../../components/Section';
+// import { IValueSection, ValueSection } from '../../components/Value';
```
I suggest keeping this uncommented and just removing the values section from the YAML file

```suggestion
import { IValueSection, ValueSection } from '../../components/Value';
```

lottamus commented on src/templates/About/index.tsx:
```diff
@@ -25,6 +28,8 @@ export interface IAbout {
   pressSection: IPressSection;
   collage: ICollage;
+  featureSection: IFeatureSection;
+  // valueSection: IValueSection;
```
Same as above

```suggestion
  valueSection: IValueSection;
```

lottamus commented on src/templates/About/index.tsx:
```diff
@@ -54,21 +59,35 @@ export const About: React.FunctionComponent<IAbout> = ({
   pressSection,
   collage,
+  featureSection,
+  // valueSection,
```
Same as above

```suggestion
  valueSection,
```

lottamus commented on src/templates/About/index.tsx:
```diff
@@ -54,21 +59,35 @@ export const About: React.FunctionComponent<IAbout> = ({
+      </Section>
+
+      {/* commenting out values section while we wait to solidify company values */}
+      {/* {valueSection.values && <ValueSection values={valueSection.values} />} */}
```
Same as above

```suggestion
      {valueSection && <ValueSection values={valueSection.values} />}
```

marbemac commented on netlify/pages/about.yaml:
```diff
@@ -1,14 +1,49 @@
+    - description: >-
+        In 2014, Marc MacLeod worked as an independent software consultant 
+        in Los Angeles building Application Programming Interfaces, known as APIs, 
+        for companies. MacLeod noticed that many of the APIs, which are systems of 
```
This sentence is quite a mouthful, something simpler? "MacLeod noticed that many of the APIs lacked documentation and were poorly designed."

paulatulis replied:
Will update!

ksatx commented:
In response to Scott's feedback about the history section, I have some thoughts. First, we could move it down, so that the features when Social Good, Careers, History, from top to bottom. Second, we could change up the content a bit but it will make me a bit to find what I'm looking for.

ksatx commented:
@paulatulis for the social good feature?


![Offset Feature Image](https://user-images.githubusercontent.com/51092436/69978384-f9de7100-14f1-11ea-9760-e993a01a0df9.png)

ksatx commented:
<img width="333" alt="Screen Shot 2019-12-02 at 2 32 21 PM" src="https://user-images.githubusercontent.com/51092436/70074100-b5b9a200-15bf-11ea-95cb-c37c8b71c78a.png">

ksatx commented:
Yes, Andoni's title needs to be updated to "Product Advocate"

ksatx commented:
![goke](https://user-images.githubusercontent.com/51092436/70079891-36ca6680-15cb-11ea-994f-750f229b7922.JPG)

paulatulis commented:
@ksatx  got 'em all except Stephen and have updated Andoni's title: 

![Screenshot 2019-12-03 at 1 44 03 PM](https://user-images.githubusercontent.com/47359669/70083959-01c21200-15d3-11ea-84f8-32966428caf7.png)

ksatx commented:
![2](https://user-images.githubusercontent.com/51092436/70096873-b5380000-15ed-11ea-9539-57b0e9d3698a.JPG)

paulatulis commented:
@ksatx done

ksatx commented:
@paulatulis can you comment out the "history" section? Then we should be good to merge, and values can wait until we have values, haha.

paulatulis commented:
@ksatx will comment out History section for now. 

Also, for link in Careers section, should it go to [AngelList](https://angel.co/company/stoplight/jobs) or to https://stoplight.io/careers?

```- description: >-
        Headquartered in Austin, Texas, our team hails from across the US and Europe, 
        and is united by our passion for APIs. From skeeball champions to rock climbers, 
        tough mudders to restaurateurs, each and every member (including our office doggies) brings their own unique 
        perspectives and experiences to the table. Thinking out of the box is our purpose 
        - the weirder the better - and challenging the status quo is our culture. Think you'd be a good fit?
        Check out our [current open roles](https://angel.co/company/stoplight/jobs), and come join us!

ksatx commented:
@paulatulis definitely [https://stoplight.io/careers](https://stoplight.io/careers) - it looks great!

paulatulis commented:
Okay, @ksatx removed history section for now, and updated careers link to go to http://localhost:3000/careers/

Let me know if there's anything else. 😄

ksatx commented:
@lottamus can you re-review, pllease? :)

paulatulis commented:
> @lottamus can you re-review, please? :)

I was trying to resolve the requested change that is blocking and I can't find it, so unless I am missing something big, I think we are good!
